youtube_transcript_api/test/test_api.py

```diff
@@ -158,7 +161,7 @@ def test_get_transcript__exception_if_no_transcript_available(self):
     def test_get_transcript__with_proxy(self):
         proxies = {'http': '', 'https:': ''}
         transcript = YouTubeTranscriptApi.get_transcript(
-            'GJLlxj_dtq8', proxies=proxies
+            'GJLlxj_dtq8', proxies=proxies, format=None
```

Comment:
all those adjustments to these tests won't be needed when `format` is removed from `get_transcript`